generate_project_files.bat

```diff
@@ -1,3 +1,4 @@
 @echo off
 call thirdparty\premake\premake5.exe --file=lumina.build.lua vs2022
 IF %ERRORLEVEL% NEQ 0 PAUSE &:: If the output code is not 0, there is probably an error
+pause
```

Comment:
does this need to stay, it currently pauses when there is an error or message

Reply:
I added this because I had a hard time reading the errors I had. But can be deleted